modules/ROOT/pages/manage-client-apps-connected-apps-concept.adoc

```diff
@@ -102,4 +97,4 @@ curl --X POST 'https://anypoint.mulesoft.com/apiplatform/repository/v2/organizat
 
 * xref:exchange::to-request-access.adoc[Request Access to API Instances from Exchange]
 * xref:remove-client-app-latest-task.adoc[Remove API Instance Contracts]
-* xref:access-client-app-id-task.adoc[Obtain Client ID and Client Secret Credentials of a Client Application]
\ No newline at end of file
+* xref:access-client-app-id-task.adoc[Obtain Client ID and Client Secret Credentials of a Client Application]
```

Comment:
Tip: if you leave the [] blank, antora will automagically fill it in with the current heading text (and you won't need to manually update it if the heading changes).